init/hive/init.sh

```diff
@@ -21,7 +21,6 @@
 sed -i '$d' /usr/local/sbin/start.sh
 sed -i '$d' /usr/local/sbin/start.sh
```

Comment:
This is the original start.sh in image: https://github.com/apache/gravitino/blob/main/dev/docker/hive/start.sh
Is it necessary to remove the last two lines.

Reply:
> Is it necessary to remove the last two lines.

I don't get it. what do you mean ?

Reply:
@xunliu @danhuawang fixed at https://github.com/apache/gravitino-playground/pull/110/commits/ed6c5b5e2ca87e0400641a328627017fcffe4edb

Reply:
I changed it to 

```
sed -i -E 's/tail -f \/dev\/null/\s/g' /usr/local/sbin/start.sh
```

, since last line might be a blank string.

fixed at 7ed9dae9db5836fd37272d3c680777972c5051b2